modtree/modtree
# fix: add/remove modules will redraw and re-CSS graph

Fixes #366 

### Summary of changes

- fix flow of calls for `removeModuleNode()` and `addModuleNode()`
- add a new graph repo method: `graphRepo.update()`. This method will probably be the final method of its class.

### Testing

- Add CS1010 to a blank graph.
- Add CS2040S. **it should be red**
- Add CS1231. **CS2040S should turn gray**
